integr8ly/delorean
# add create-release command

You can see an example PR created by this command: https://github.com/wei-lee/integreatly-operator/pull/5.

To verify:

1. Fork the integr8ly/integreatly-operator repo if you haven't done so. Create a new branch that is based off this one: https://github.com/StevenTobin/integreatly-operator/tree/DEL-219.
2. Generate a github personal access token if you haven't done so.
3. Set the following env vars (best to create a `.delorean.yaml` config file in your home dir and add the following env vars there):
    ```
    export GITHUB_USER=<your github username>
    export GITHUB_TOKEN=<your github access token>
    ```
4. Run the command:
    ```
     ./delorean release create-release --owner <your-fork> --repo integreatly-operator --baseBranch <your-base-branch> --version 2.2.0-rc1
    ```
5. You should see some output and when the command finished, a PR is created.
